platform/linux-dpdk/Makefile.am

```diff
@@ -196,6 +197,7 @@ noinst_HEADERS = \
 		  ${top_srcdir}/platform/linux-generic/include/odp_packet_socket.h \
 		  ${top_srcdir}/platform/linux-generic/include/odp_pkt_queue_internal.h \
 		  ${srcdir}/include/odp_pool_internal.h \
+		  ${top_srcdir}/platform/linux-generic/include/odp_pool_subsystem.h \
```

Comment:
@nagarahalli what do you mean by "splitting already and moving them now", copy the odp_pool_subsystem into linux-dpdk ?